platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -1635,6 +1645,16 @@
 			path = ../vendor/SMCalloutView;
 			sourceTree = "<group>";
 		};
+		DA8848911CBB049300AB86E3 /* reachability */ = {
```

Comment:
These reachability references were removed in #10940 in `master` (and don’t need to be added back).